src/components/Instructions.component.js

```diff
@@ -1,18 +1,36 @@
 import React, { Component } from 'react'
-// TODO: uncomment this for Part 1
-// import PropTypes from 'prop-types'
+import PropTypes from 'prop-types'
 import './../styles/instructions.css'
 
 class Instructions extends Component {
+  static propTypes = {
+    shouldDisplayImage: PropTypes.bool,
+    items: PropTypes.arrayOf(PropTypes.string),
+  }
   render() {
     return (
       <div className="instructions">
         Follow the instructions on the README to get started!
-        <img
+        {this.props.shouldDisplayImage ? (
+          <img
           className="instructions__logo"
           src="https://uiuc.hack4impact.org/img/colored-logo.png"
           alt="h4i logo"
-        />
+          />
+        ) : (
+          <div />
+        )}
+        <ul>
+          {this.props.items.filter(function(item){
+            return item.length >= 3
+          }).map((item, index) => item.length < 3 ? (
+            <div key={index}/>
```

Comment:
dont use nested ternary operators, but more importantly, you shouldn't need to check the `item.length` here since you already checked in the `filter` above